docs/jwst/outlier_detection/outlier_detection_ifu.rst

```diff
@@ -14,7 +14,9 @@ in the :ref:`calwebb_detector1 <calwebb_detector1>` pipeline.
 At this time it is believed that NIRSpec IFU data also have bad pixels that vary with time, though the time variation is
 still under study.
 
-An algorithm was developed to flag pixels that are  outliers when compared to their neighbors for a set of
+The basic of that the idea of the outlier detection plan for IFU data  is to look for pixels on the detector
```

Comment:
"... basic of that the idea ..." really needs some cleanup. Perhaps "The basis of the outlier detection plan for IFU data ..."